Packs/CommonPlaybooks/Playbooks/playbook-URL_Enrichment_-_Generic_v2_README.md

```diff
@@ -6,6 +6,10 @@ URL enrichment includes:
 * Providing of URL screenshots.
 * URL Reputation using !url.
 
+## Limitation
+
+The "Get URL screenshot" task has a timeout of 10 minutes when using the Rasterize integration. This allows processing a large number of URLs. If the task still fails due to timeout because the Rasterize command is processing too many URLs, the timeout value of the task needs to be increased.
```

Comment:
This allows you to process a large number of URLs. If the task still fails due to timeout because the Rasterize command is processing too many URLs, increase the timeout value of the task.